recipes/snipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "snipe" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/snipe-bio/snipe/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a25432955f992119c8ce69e38fb03e7d79eff8c2880398944b61b0b628fd7d01
+
+
+entry_points:
```

Comment:
This should be under `build` - https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#python-entry-points

Reply:
Fixed; Thanks!